CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Next Release
 * [#1045](https://github.com/intridea/grape/pull/1045): Do not convert `Rack::Response` to `Rack::Response` in middleware - [@dmitry](https://github.com/dmitry).
 * [#1048](https://github.com/intridea/grape/pull/1048): Only dup `InheritableValues`, remove support for `deep_dup` - [@toddmazierski](https://github.com/toddmazierski/)
 * [#1052](https://github.com/intridea/grape/pull/1052): Reset `description[:params]` when resetting validations - [@marshall-lee](https://github.com/marshall-lee).
+* [#1065](https://github.com/intridea/grape/pull/1065): Avoid object allocation in util/stackable_values.rb and http/request.rb - [@markgeek](https://github.com/markgeek)
```

Comment:
This is a bit of an internal thing, I don't think we need a changelog line. If you think we do I am totally ok leaving it, but then make it look like other changelog entries (change util/stackable_values.rb to the class name and add a period at the end). Also, it's not like StackableValues avoids allocation altogether, I think it should say "Reduce object allocation ...".